### PR TITLE
Rover: Adding a class name

### PR DESCRIPTION
**Pull request by muramura**
Except for this method, all other methods are referenced by their class names.
Referenced with class name to match other methods.
I want to be consistent within the file.

### CODE REVIEW

**Review by khancyr — APPROVED** [review]
Just code style change.

**Comment by khancyr** [comment]
@rmackay9 As you are the maintainer, I leave this to your call as it is just code style, so we merge or close.

**Comment by rmackay9** [comment]
Hi, I don't think I'll merge this actually.  It's not bad but it's not really an improvement either so I tend towards reducing "code churn"

**Pull request closed without merging**
